src/univers/versions.py

```diff
@@ -447,6 +446,33 @@ def build_value(cls, string):
     def __str__(self):
         return f"{self.value[0]}.{self.value[1]}.{self.value[2]}{self.value[3]}"
 
+    def __lt__(self, other):
+        if not isinstance(other, self.__class__):
+            return NotImplemented
+        base_vers1 = self.value[:-1]
```

Comment:
Rather than having a positional tuple, why not use instead a proper object for this? an attr class with major, minor, build, and patch attributes would go a long way and would avoid having to deal with the magic [:-1]?
Or more simply, just define major, minor, build, and patch as attributes on this object itself. This would be much more explicit and cleaner IMHO.... Always prefer names over positional indexes.